### DISCUSSION TITLE
Any way to turn off the "vehicle occupied" feature?

### DISCUSSION THREAD

**Discussion by DerpyHoowes** [SmartlyDressedGames/Unturned-3.x-Community#3991] [Q&A] [answered]
When a player is sitting in a vehicle, you cant place barricades on it, with a message appearing that "Vehicle occupied". Is there any asset tag to turn it off, so that the barricades can be placed on an occupied vehicle too?
![2023-07-04_01-18-06](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/59206749/8838a61d-91b2-45be-a5a8-37cf9a34c4f5)

**Comment by SDGNelson** [comment, marked as answer]
Battlefield style blowing up the enemy tank! Unfortunately I think that restriction is because there were problems with the steering getting locked if the rigidbody's colliders changed while driving. Maybe resolved in a future engine version 🤷‍♂️
